DXMainClient/Domain/Multiplayer/MultiplayerColor.cs

```diff
@@ -59,7 +59,9 @@ public static List<MultiplayerColor> LoadColors()
 
                 try
                 {
+#pragma warning disable CNCNET0001
```

Comment:
I'd suggest to instead make it a suggestion.

Reply:
Replied